.docker/ckan.ini

```diff
@@ -227,6 +227,7 @@ ckan.comments.show_comments_tab_page = True
 # Archiver
 ckanext-archiver.archive_dir = /app/filestore/archive
 ckanext-archiver.cache_url_root = http://dataqld-ckan.docker.amazee.io/resources/
+ckanext-archiver.user_agent_string =
```

Comment:
for dev testing that is ok, but we should be setting this in our environment to a browser ua string + site.

I.e.
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36 CKAN

Reply:
The current setting is `ckanext-archiver.user_agent_string = "CKAN archiver https://<%= @app_url %><% node['datashades']['ckan_web']['endpoint'] %>"`

Eg `CKAN archiver https://dev.data.qld.gov.au`

Reply:
This is fine if we are not being blocked.